crates/node_runtime/src/node_runtime.rs

```diff
@@ -220,34 +225,85 @@ impl NodeRuntime {
             .npm_package_installed_version(local_package_directory, name)
             .await
     }
-
     pub async fn npm_package_latest_version(&self, name: &str) -> Result<String> {
-        let http = self.0.lock().await.http.clone();
-        let output = self
-            .instance()
-            .await
-            .run_npm_subcommand(
-                None,
-                http.proxy(),
-                "info",
-                &[
-                    name,
-                    "--json",
-                    "--fetch-retry-mintimeout",
-                    "2000",
-                    "--fetch-retry-maxtimeout",
-                    "5000",
-                    "--fetch-timeout",
-                    "5000",
-                ],
-            )
-            .await?;
+        let instance = self.instance().await;
+        match instance.runtime_type() {
+            RuntimeType::Node => {
+                let http = self.0.lock().await.http.clone();
+                let output = self
+                    .instance()
+                    .await
+                    .run_npm_subcommand(
+                        None,
+                        http.proxy(),
+                        "info",
+                        &[
+                            name,
```

Comment:
I didn't notice this the first time. If it is possible for a package to start with `-` or `--` this could break the command invocation. It would have to be `<cmd> <args...> -- <packages...>`. This applies to a couple of other places too.

Reply:
Might be out of scope of this PR since it was like that before.